docs/jwst/associations/level2_asn_technical.rst

```diff
@@ -29,75 +27,79 @@ Example Association
 The following example will be used to explain the contents of an association::
 
     {
-        "asn_rule": "Asn_Lv2Spec",
-        "asn_pool": "jw82600_001_20160304T145416_pool",
-        "program": "82600",
-        "asn_type": "spec2",
+        "asn_type": "image2",
+        "asn_rule": "candidate_Asn_Lv2Image",
+        "version_id": "20210610t121508",
+        "code_version": "1.2.3",
+        "degraded_status": "No known degraded exposures in association.",
+        "program": "00623",
+        "constraints": "DMSAttrConstraint({'name': 'program', 'sources': ['program'], 'value': '623'})\nDMSAttrConstraint({'name': 'is_tso', 'sources': ['tsovisit'], 'value': None})\nDMSAttrConstraint({'name': 'instrument', 'sources': ['instrume'], 'value': 'miri'})\nDMSAttrConstraint({'name': 'detector', 'sources': ['detector'], 'value': 'mirimage'})\nDMSAttrConstraint({'name': 'opt_elem', 'sources': ['filter'], 'value': 'f1130w'})\nDMSAttrConstraint({'name': 'opt_elem2', 'sources': ['pupil', 'grating'], 'value': None})\nDMSAttrConstraint({'name': 'opt_elem3', 'sources': ['fxd_slit'], 'value': None})\nDMSAttrConstraint({'name': 'subarray', 'sources': ['subarray'], 'value': 'brightsky'})\nDMSAttrConstraint({'name': 'channel', 'sources': ['channel'], 'value': None})\nDMSAttrConstraint({'name': 'slit', 'sources': ['fxd_slit'], 'value': None})\nConstraint_Image_Science({'name': 'exp_type', 'sources': ['exp_type'], 'value': 'mir_image'})\nConstraint_Single_Science({'name': 'single_science', 'value': False})\nDMSAttrConstraint({'name': 'asn_candidate', 'sources': ['asn_candidate'], 'value': \"\\\\('o037',\\\\ 'observation'\\\\)\"})",
+        "asn_id": "o037",
+        "asn_pool": "jw00623_20210610t121508_pool",
+        "target": "9",
```

Comment:
Do `target` numbers really show up in level-2b ASN files? Since everything is exposure-centric at this level, I wouldn't think target would be relevant.

Reply:
I know of one explicit case where it is necessary, Asn_Lv2WFSS. However, as being part of the meta information, it is just informational and could be removed.